app/actions/remote/channel.ts

```diff
@@ -46,6 +46,32 @@ export type MyChannelsRequest = {
     error?: unknown;
 }
 
+export async function removeMembersFromChannel(serverUrl: string, channelId: string, userIds: string[]) {
+    const operator = DatabaseManager.serverDatabases[serverUrl]?.operator;
+    if (!operator) {
+        return {error: `${serverUrl} database not found`};
+    }
+
+    let client: Client;
+    try {
+        client = NetworkManager.getClient(serverUrl);
+    } catch (error) {
+        return {error};
+    }
+
+    try {
+        const promises = userIds.map((id) => client.removeFromChannel(id, channelId));
+        await Promise.all(promises);
+
+        const modelPromises = userIds.map((id) => deleteChannelMembership(operator, id, channelId));
+        await Promise.all(modelPromises);
+        return {error: undefined};
+    } catch (error) {
+        forceLogoutIfNecessary(serverUrl, error as ClientErrorProps);
+        return {error};
+    }
```

Comment:
As the design currently stands, you can only remove one member at a time.  I modified this function to accept only a single user. If we add the ability to select multiple and remove multiple users, we'll add it then.

We should also consider a server API that removes mulitiple users so we don't have to make N number of client calls instead of one.